internal/pkg/service/common/task/node_test.go

```diff
@@ -46,7 +46,7 @@ func TestSuccessfulTask(t *testing.T) {
 	}
 
 	logs := ioutil.NewAtomicWriter()
-	tel := telemetry.NewForTest(t)
+	tel := newTestTelemetryWithFilter(t)
 
 	// Create nodes
 	node1, _ := createNode(t, etcdCredentials, logs, tel, "node1")
```

Comment:
Tu bol este problem v testoch.

`createNode` -> vola `createDeps` a tam sa volalo `tel.SetSpanFilter(...)`.
- teda ked tu mame v test 2 nodes, tak sa to volalo duplicitne
- okrem toho `SetSpanFilter` vola `Reset` nad telemetriou
- a to ^^^ bol hlavny problem, jedna node - `node1` uz bezala, `node2` sa setupovala, a zavolal sa `tel.Reset()`.
- `node1` mohla uz vytvorit nejake spans, nie je definovane poradie operacii, nie je tam zamok

Fix:
- `SetSpanFilter` sa zavola iba raz, a to pred vytvorenim vsetkych nodes


----


![image](https://github.com/keboola/keboola-as-code/assets/19371734/111572d1-6e18-4222-b3be-5be6382f9cba)